src/lib/mdns/Discovery_ImplPlatform.cpp

```diff
@@ -477,6 +477,8 @@ void DiscoveryImplPlatform::HandleNodeResolve(void * context, MdnsService * resu
         data.ipAddress[data.numIPs++] = result->mAddress.Value();
     }
 
+    data.port = result->mPort;
```

Comment:
Need to store the interface id somewhere here too.

Reply:
Done.